libbeat/processors/rate_limit/docs/rate_limit.asciidoc

```diff
@@ -0,0 +1,39 @@
+[[rate_limit]]
+=== Rate limit the flow of events
+
```

Comment:
Make it beta by now, in case we need to change something breaking.
```suggestion
beta[]
```

Reply:
Done in 12af0cecff8d4df4da5c04a0cf8e1df006bdffb6.